### DISCUSSION TITLE
Discuss Public Utility Data Liberation with us!

### DISCUSSION THREAD

**Discussion by zaneselvans** [catalyst-cooperative/pudl#1569] [Announcements]
## 👋 Welcome!
We’re enabling Discussions here on the main PUDL repository as an easier point of entry to interacting with the project. Sometimes folks seem reluctant to open issues to get their questions answered, but there's a lot of value in having questions and discussion happening in a public (searchable!) location so that people who come along later with the same questions can benefit.

Feel free to use Discussions to:
  * Ask questions about how to set up and use PUDL and the data it contains.
  * Share ideas about new datasets and analyses that you'd like to see incorporated.
  * Find other people working with open energy data for advocacy and research purposes.
  * Share your own PUDL use cases! We'd love to know more about how this data is being used.

If you've been lurking in the repo for a while and feel like introducing yourself, leave a comment below!

**Comment by MichaelTiemannOSC** [comment]
I've been aware of PUDL for nearly a year, and it was love at first sight.  I mostly use PUDL data as surfaced in the RMI Utilities Transition Hub datasets, but find it extremely helpful to use Datasette to drill down into PUDL's primary data.

Looking forward to helping augment PUDL with better entity-matching and share experiences building unit-aware data frames with Pint and Pint-Pandas.

I also see value in cross-checking EIA and FERC data with EPA GHGRP as well as ESSD tops-down emissions.  Very interested in how or whether PUDL data might tie up with EEI Scope 2 (not only Scope 1) emissions reporting.

Michael Tiemann
OS Climate Project Lead
